web/policies/organization_policy.ex

```diff
@@ -8,6 +8,7 @@ defmodule CodeCorps.OrganizationPolicy do
   def create?(%User{admin: true}), do: true
   def create?(%User{admin: false}), do: false
 
+  def update?(%User{admin: true}, %Organization{}), do: true
   def update?(%User{} = user, %Organization{} = organization), do: user |> role_is_at_least_admin(organization)
```

Comment:
That should never happen as something else really major would have failed in that event (i.e. there's no current user even in the `conn` to be able to check against.